project/api/Company.Api/Functions/GuestFunction.cs

```diff
@@ -73,7 +73,10 @@ public async Task<IActionResult> GetGuestByIdAsync([HttpTrigger(AuthorizationLev
     {
         try
         {
-            var response = await _cmsService.GetGuestByIdAsync(Guid.Parse(id));
+            var res = Guid.TryParse(id, out var parsedGuid);
```

Comment:
Could call the variable parsed/isParsed instead of res. There are more like this.